src/extensions/default/InlineColorEditor/css/main.less

```diff
@@ -56,6 +56,9 @@
     position: relative;
     overflow: hidden;
     -webkit-background-clip: padding-box;
+    -moz-background-clip: padding-box;
+    -o-background-clip: padding-box;
+    -ms-background-clip: padding-box;
```

Comment:
http://caniuse.com/#search=background-clip and https://developer.mozilla.org/en-US/docs/Web/CSS/background-clip and http://shouldiprefix.com/#background-image-options all seem hopeful about not needing prefixes on this.